ui/src/css/helpers/math.sass

```diff
@@ -21,22 +21,22 @@ $PI: 3.14159265359
 // toFixed(0.12345, 100) -> 0.12
 // toFixed(0.12345, 1000) -> 0.123
 @function toFixed($number, $power)
-  @return round($number * $power) / $power
+  @return calc(round($number * $power) / $power)
```

Comment:
Seems to me this would be better off written as it should be (instead of using `calc`):
```css
@return round(math.div($number * $power, $power))
```

Reply:
great, thanks!